dependencies.list

```diff
@@ -1,7 +1,7 @@
 # Realm Sync Core release used by Realm Java
 # https://github.com/realm/realm-sync/releases
-REALM_SYNC_VERSION=1.8.5
-REALM_SYNC_SHA256=71e70f83b1604672bbfd7c04fcf984ecc8ac683864943ef3de262994c2e7b7fc
+REALM_SYNC_VERSION=1.9.1
```

Comment:
I bumped the commit to the latest possible. It seems that OS was never built against 1.9.1, but there is no breaking changes so It should be okay.